hack/make-rules/verify.sh

```diff
@@ -220,7 +220,7 @@ fi
 
 ret=0
 run-checks "${KUBE_ROOT}/hack/verify-*.sh" bash
-run-checks "${KUBE_ROOT}/hack/verify-*.py" python
+run-checks "${KUBE_ROOT}/hack/verify-*.py" python3
```

Comment:
Kinda curious why ${KUBE_ROOT} is `hack/make-rules/../../`, doesn't impact anything but just odd imo.